src/lib/telemetry/TelemetryStore.ts

```diff
@@ -26,7 +28,11 @@ export class TelemetryLocalStore implements ITelemetryLocalStore {
   }
 
   public getMetrics() {
-    return this.store.get('metrics') || []
+    const metrics = this.store.get('metrics')
+    if (!isArray(metrics)) {
```

Comment:
Oops, sorry, just saw that `isArray` from utils is deprecated, we'll have to use `Array.isArray`